src/interactive/cli.ts

```diff
@@ -361,8 +361,76 @@ export async function interactiveCli(baseargv: string[]) {
       }
     }
 
+  else if (Object.keys(taskConstants)[11] == task.toString()) {
+    const importDestChain = await prompts.importTrxType()
+    let trxType;
+    if (importDestChain.type == 'P')
+      trxType = 'CP'
+    if (importDestChain.type == 'C')
+      trxType = 'PC'
+
+    if (walletProperties.wallet == Object.keys(walletConstants)[0] && fileExists("ctx.json")) {
+      const { network: ctxNetwork, derivationPath: ctxDerivationPath } = readInfoFromCtx("ctx.json")
+      if (ctxNetwork && ctxDerivationPath) {
+        const argsImport = [...baseargv.slice(0, 2), "transaction", `import${trxType}`, "--blind", "true", "--derivation-path", ctxDerivationPath, `--network=${ctxNetwork}`, "--ledger"]
+        // ask for fees if its importTxPC
+        if (importDestChain.type == 'C') {
```

Comment:
what about the case of p import transaction ? how you handle if user pass lowercase p or c

Reply:
Its a drop down not something user manually imports